lib/northbound_cli.c

```diff
@@ -1724,6 +1724,7 @@ void nb_cli_init(struct thread_master *tm)
 
 	/* Install debug commands */
 	debug_init(&nb_dbg_cbs);
+	debug_init_cli();
```

Comment:
nit: I'd prefer to call this from `frr_init()`, since the northbound and the new debug infrastructure are not directly related.

Reply:
I think this would be a good change...

Reply:
+1

Reply:
fixed